plugin/src/main/groovy/com/novoda/gradle/command/MonkeySpec.groovy

```diff
@@ -0,0 +1,37 @@
+package com.novoda.gradle.command;
+
+final class MonkeySpec {
+
+    private static final int EVENTS_DEFAULT = 10000
+
+    def events
+    def seed
+    def categories = []
+
+    void events(events) {
+        this.events = events
+    }
+
+    void seed(seed) {
+        this.seed = seed
+    }
+
+    void categories(... categories) {
+        this.categories.addAll(categories)
+    }
+
+    // prefer system property over direct setting to enable commandline arguments
+    def getEvents() {
+        System.properties['events'] ?: events ?: EVENTS_DEFAULT
+    }
+
+    // prefer system property over direct setting to enable commandline arguments
+    def getCategories() {
+        def systemCategories = System.properties['categories']
+        systemCategories ? [systemCategories] : categories
```

Comment:
Here this also fixes a problem with command line argument support. 
Basically command line support for categories was never working. 

This command `./gradlew monkeyFreeBetaDebug -i -Dcategories=test` would resolve into this output

```
[adb, -s, 02c2095608e77be5, shell, monkey, -p, com.novoda.gradle.command, -c, t, -c, e, -c, s, -c, t, -v, 1000]
```

It basically splits the given String into pieces. 
Now only when the command line argument is provided, I am putting it into an array.